bleak/_api.py

```diff
@@ -0,0 +1,237 @@
+import os
+import platform
+from bleak.exc import BleakError
+from bleak.backends.scanner import BaseBleakScanner
+from bleak.backends.client import BaseBleakClient
+
+
+_on_rtd = os.environ.get("READTHEDOCS") == "True"
```

Comment:
If this is only used once, we probably just inline it instead of using a variable.

Reply:
It is now used twice:-)